client/src/components/card/ReviewCard.jsx

```
@@ -1,7 +1,234 @@
+    <div className={`${styles.card} ${toggleCard && styles.flipped}`}>
+      <div className={styles.cardFront}>
+        <div className={styles.topWrapper}>
+          <div className={styles.imageWrapper}>
+            <img src={reviewImageUrl} alt="cardImage" />
+            <div className={styles.bookIconWrapper}>
+              <img src={bookIcon} alt="bookIcon" className={styles.bookIcon} />
+              <div className={styles.bookGrayText} onClick={toggleCardHandler}>
+                책 정보 보기
+              </div>
+            </div>
+          </div>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)